layouts/index.html

```diff
@@ -15,15 +15,23 @@ <h1>{{ (index $.Site.Params.lang_strings .Site.Language.Lang).home_title }}</h1>
     {{ range .Site.Menus.news }}
     <h2>{{ .Name }}</h2>
     {{ if .Post }}<p class="small text-muted">{{ .Post | markdownify}}</p>{{ end }}
+    {{ end }}
+    {{ $news_high := where .Site.RegularPages "Section" "news" }}
+    {{ if eq $.Site.Language.LanguageName "Svenska" }}
+    {{ $display_alert := "false" }}
+    {{ range first 3 $news_high }}{{ if .Params.needs_translation }} {{$display_alert := "true" }} {{ end }} {{ end }}
+    {{ if $display_alert}}<div class="alert alert-info">
+      <i class="fas fa-exclamation-triangle"></i>
+      <span class="small">OBS. I några fall är svensk översättning snart tillgänglig.</span>
```

Comment:
Not completely happy wtih the wording "OBS. I några fall är svensk översättning snart tillgänglig."

Maybe "Svensk översättning för de engelska texterna kommer inom kort."?